provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -67,8 +67,12 @@ def _create_node_descriptions(self) -> List[NodeDesc]:
         machine_ids = conf.get_machine_ids_for_service(
             Const.SERVICE_MOTR_IO.value)
 
-        machine_ids.extend(conf.get_machine_ids_for_service(
-            Const.SERVICE_S3_SERVER.value))
+        s3_nodes = conf.get_machine_ids_for_service(
```

Comment:
Let the earlier implementation be, just do,
machine_ids = list(set(machine_ids)) -> removes duplicates.

Reply:
Done